OpenOversight/app/templates/partials/officer_faces.html

```diff
@@ -18,7 +18,7 @@
       </div>
     {% endfor %}
   </div>
-  {% if paths | length > 1 %}
+  {% if face_paths | length > 1 %}
```

Comment:
Could you show a screenshot of this, to show it works?

Reply:
Yup!

Before:
![1](https://github.com/lucyparsons/OpenOversight/assets/66500457/c1116cd5-9b24-45e1-877f-bf5f76537b08)


After:
![2](https://github.com/lucyparsons/OpenOversight/assets/66500457/325c8935-77fb-42cb-927f-d10aa4133662)

Reply:
That's a round bb. Appreciate it.